<ui version="4.0" >
 <class>SinerjiGui</class>
 <widget class="QDialog" name="SinerjiGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sinerji</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="4" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="1" >
       <widget class="QComboBox" name="topComboBox" />
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="leftComboBox" />
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="rightComboBox" />
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="bottomComboBox" />
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="icobLabel" >
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap>images/icon.png</pixmap>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="applyButton" >
     <property name="text" >
      <string>Apply</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>images/buttonOk.png</normaloff>images/buttonOk.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>Close</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>images/buttonCancel.png</normaloff>images/buttonCancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
